File: Year 2/Semester 4/MPP/REST/RESTJava/Persistance/src/main/java/music/rest/persistance/repository/ITicketRepository.java
package music.rest.persistance.repository;

import rest.domain.Entity;
import rest.domain.Ticket;

import java.io.Serializable;

public interface ITicketRepository <ID extends Serializable, E extends Entity<ID>> extends IRepository<ID, E> {
    void addTicket(Ticket ticket);
}

File: Year 2/Semester 4/Programming and Projection Environments/Week 4/C#FourthWeek/SaptamanaAPatra/repository/interfaces/ITicketRepository.cs
using SaptamanaAPatra.domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SaptamanaAPatra.repository.interfaces
{
    internal interface ITicketRepository : IRepository<int, Ticket<int>>
    {
    }
}
